### ISSUE TITLE
Make K8s production checklist dashboard

### ISSUE THREAD

**Issue by cristiklein** [elastisys/compliantkubernetes-apps#322]
**What should be investigated.**

The [Kubernetes production checklist](https://learnk8s.io/production-best-practices) is an every-growing list of best-practices for making both the Kubernetes clusters and the hosted application production-ready.

I did a quick evaluation and 50% of the items on the checklist fall into the following groups:
1. They can already be evaluated via the Compliant Kubernetes dashboards.
2. They cannot be evaluated via dashboards (e.g., "The app doesn't shut down on SIGTERM, but it gracefully terminates connections.").

It would make sense to investigate if we can produce dashboards to help evaluate the remaining 50%.

**What artifacts should this produce.**

List which goes systematically through the checklist and specifies:
* How Compliant Kubernetes helps evaluate an item *today* (e.g., link to documentation).
* How Compliant Kubernetes could help evaluate an item *in the future*. Please make a T-shirt sized estimation.
* Why Compliant Kubernetes could not help evaluate an item via a dashboard.

**Comment by viktor-f** [comment]
Would be nice to also gather all of the checklist items (at least everything we can show in dashboards) into one single dashboard, instead of having to look through several. (as the issue title suggests)

**Comment by cristiklein** [comment]
Agreed. We will likely have to produce some "meta-dashboards" for K8s production checklist, ISO 27001, etc.

**Comment by tordsson** [comment]
I see a need for an overall rehaul of all dashboards, to better capture the workflow of ck8s users. Which roles? Which problem, which information?
